examples/xgboost_penguins/__init__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2021 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Are licenses required for `__init__.py` files?

Reply:
Yes, with the current configuration of the lint check.